### PR TITLE
Write reformatDates method.

### PR DESCRIPTION
**Pull request by Arkarine**
#7

### CODE REVIEW

File: DateTimeUsing/src/DateFormatModifier.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            dates.add(parse);
+        }
+
+        List<String> result = new ArrayList<>();
+
+        for (int j = 0; j < dates.size(); j++) {
+            String format = dates.get(j).format(DateTimeFormatter.ofPattern("(dd. MMM, yyyy EEEE)"));
+            result.add(format);
+        }
+        return result;
```
Comment (sergeylukichev) [inline DateTimeUsing/src/DateFormatModifier.java:63]: can you make it all in one line using Stream API?

Comment (Arkarine) [reply DateTimeUsing/src/DateFormatModifier.java:63]: The task is done.

Comment (sergeylukichev) [reply DateTimeUsing/src/DateFormatModifier.java:63]: I am not sure you pushed the changes.

Comment (Arkarine) [reply DateTimeUsing/src/DateFormatModifier.java:63]: Please find them in this commit https://github.com/Arkarine/Homeworks/pull/8/commits/da3089f0e35c998f038ef81d2a30fe32b4c89f71  

        List<String>  reformattedDates = list
                .stream()
                .map(s -> LocalDate.parse(s))
                .map(date -> date.format(DateTimeFormatter.ofPattern("(dd. MMM, yyyy EEEE)")))
                .collect(Collectors.toList());

        System.out.println(reformattedDates);

Comment (sergeylukichev) [reply DateTimeUsing/src/DateFormatModifier.java:63]: Good, thanks.

**Pull request merged**